lib/parse.js

```diff
@@ -335,6 +343,85 @@ class Parser {
         if (matches(token, '#') || isIdentifierToken(token)) {
             const node = this.identifierOrPlaceholder()
 
+            const r1 = new RegExp(derivativeNumeratorPattern);
+            const r2 = new RegExp(derivativeDenominatorPattern);
+
+            // Lagrange notation
+            // e.g. f', f'', f'(x)
+            if (node.name === 'f' && matches(this.currentToken(), `'`)) {
```

Comment:
This is pretty restrictive.  We should be able to parse the derivative of any function (or expression).  This is so we can handle things like `(af + bg)'`.